packages/desktop_security_center/lib/src/l10n/app_de.arb

```diff
@@ -0,0 +1,37 @@
+{
+    "appTitle": "Desktop Security Center",
+    "ubuntuPro": "Ubuntu Pro",
+    "ubuntuProEnabled": "Ubuntu Pro ist aktiviert",
+    "ubuntuProDisabled": "Nicht geschützt mit Ubuntu Pro",
+    "getUbuntuPro": "Ubuntu Pro erhalten",
+    "attachThisMachine": "Diese Maschine anbinden",
+    "ubuntuProFreeDesc": "Immer kostenlos für den persönlichen Gebrauch.\n30-Tage-Testversion für Unternehmen.",
+    "esmLabel": "Erweiterte Sicherheitswartung (ESM)",
+    "esmDesc": "ESM bietet zusätzliche Sicherheitspatches für 10 Jahre für mehr als 23.000 Open-Source-Pakete. Es ermöglicht ein kontinuierliches Schwachstellenmanagement für kritische, hohe und mittlere CVEs.",
+    "proKernelLivepatchLabel": "Kernel-Livepatch",
+    "proEsmMainPackages": "Hauptpakete (esm-infra)",
```

Comment:
As main is the name of the repository, I'd not translate that word and write it as you did in the line below (Main-Pakete).

Reply:
Yes, it would be better not to translate the name of the repository. I have changed it.